invenio_rdm_records/services/components/pids.py

```diff
@@ -9,7 +9,7 @@
 
 """RDM service component for pids."""
 
-import re
+import logging
```

Comment:
This is probably my fault 😅 but I think we should use `current_app.logger` instead, that way it can go to e.g. Sentry